.github/workflows/integration-tests.yml

```diff
@@ -19,28 +19,19 @@ env:
   REPO: ${{ github.repository_owner }}
 jobs:
   test:
-    runs-on: runs-on,runner=8cpu-linux-x64,image=legacy-cgroups-for-x64,run-id=${{ github.run_id }}
+    runs-on: runs-on,runner=16cpu-linux-x64,ram=64,run-id=${{ github.run_id }}
     timeout-minutes: 60
     env:
       K3D_VERSION: v5.7.1
     steps:
       - name: Checkout code
         uses: actions/checkout@v4
+      - name: Configure Docker for cgroupfs
+        run: |
+          echo '{"exec-opts": ["native.cgroupdriver=cgroupfs"]}'| sudo tee /etc/docker/daemon.json
+          sudo systemctl restart docker
       - name: Setup Environment Variables
         uses: ./.github/actions/setup-tag-env
-      - name: Clean up Docker
-        run: |
-          docker system prune -af
```

Comment:
Where these doing anything?

Reply:
I think this step was part of a cleanup that was needed to free up space on old runners